README.md

```diff
@@ -1,14 +1,26 @@
 # Portfolio-mobile-version
 Mobile version
```

Comment:
Update your README file and provide missing features. This is a [template from Microverse](https://github.com/microverseinc/readme-template) i hope that is functional for you.